### PR TITLE
Add parsing of component versions

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@flopezag 
#### Description

Add parsing of 'region.pool_ip' metadata including the services versions.
#### Testing

Tests passed

### CODE REVIEW

File: monasca_persister/README.rst [hunk 1]
```diff
@@ -46,45 +46,63 @@ Those data points are mapped to NGSI Entities according to the following:
...
-   * - region.allocated_ip
+   * - region.info
      - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
-     - region.allocated_ip
+       | {component}_version
```
Comment (flopezag) [inline monasca_persister/README.rst]: food for your mind, the number of OpenStack services that are deployed in one region is a no fixed number and can be increased or decreased in time. In fact take the example of Spain2 node.

File: monasca_persister/README.rst [hunk 2]
```diff
@@ -46,45 +46,63 @@ Those data points are mapped to NGSI Entities according to the following:
...
      - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
+       | {component}_version
      - region.used_ip
      - |
        |
```
Comment (flopezag) [inline monasca_persister/README.rst:95]: I would like to add in the documentarion that the {component}_version will follow the data in [http://releases.openstack.org/].

Hopefully I hope...

File: monasca_persister/README.rst (hunk 1 above)
Comment (pratid) [inline monasca_persister/README.rst]: Nothing about this component suggests such fixed number of services.

File: monasca_persister/README.rst [hunk 3]
```diff
@@ -138,20 +146,21 @@ file.
...
-     - *[none]*
-     - *component* (\*)
+     - *none*
+     - *component* [\*]
 
-(*)
-A 'component' dimension will identify the exact process (e.g. `nova-api`) part
-of an OpenStack service (given by the 'service' dimension, e.g. `nova`).
+**[\*]**
+The 'component' dimension will identify the exact process (for instance,
```
Comment (flopezag) [inline monasca_persister/README.rst]: identifies, not the future...

**Comment by flopezag** [comment]
LGTM

**Pull request merged**